bears/vcs/git/GitCommitBear.py

```diff
@@ -31,6 +31,18 @@ class GitCommitBear(GlobalBear):
     }
     CONCATENATION_KEYWORDS = [r',', r'\sand\s']
 
+    _nltk_data_downloaded = False
```

Comment:
This variable is only useful for the tests, right? If that's true, I'd recommend removing it entirely and instead use `mock.patch` to check if `nltk.download` was called in the test.